controllers/poto.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const {ValidationError} = require('sequelize');
+
+const router = express.Router();
+const responseUtil = require('../helpers/response');
+const { Photo, User ,Comment} = require('../models');
```

Comment:
ini kalau ga dipake dihapus aja yaa,, coba dihapus dulu nanti
```suggestion
const { Photo } = require('../models');
```